### PR TITLE
adding supporting for gtdlocation

### PR DESCRIPTION
**Pull request by ashmere**
@miekg can you work with me on getting this to a PR state

### CODE REVIEW

**Comment by miekg** [comment]
this looks good to me, but you probably need add docs and tests as well.

**Comment by miekg** [comment]
@ashmere willing to help, but what do you want/need?

**Comment by ashmere** [comment]
@miekg I believe this is in a good state now. added tests and docs.
Also tested with a non-enabled gtd domain that sending gtdLocation = "DEFAULT" causes no problem
if it is set to a region you will get a 400 from the api

File: builtin/providers/dme/resource_dme_record_test.go [hunk 1]
```diff
@@ -36,6 +36,9 @@ func TestAccDMERecord_basic(t *testing.T) {
 						"dme_record.test", "value", "1.1.1.1"),
 					resource.TestCheckResourceAttr(
 						"dme_record.test", "ttl", "2000"),
+                                        resource.TestCheckResourceAttr(
+                                                "dme_record.test", "gtdLocation", "DEFAULT"),
```
Comment (miekg) [inline builtin/providers/dme/resource_dme_record_test.go:40]: you only test it with 'DEFAULT'? Is there a testcase that you can make where you actually to something more useful?

**Comment by miekg** [comment]
LGTM

**Comment by ashmere** [comment]
closing as I'll make the PR on the squashed branch b-dme-support-gtdlocation

**Pull request closed without merging**